docs/roadmap.md

```diff
@@ -53,80 +42,59 @@ It focuses on enabling three main use cases:
 2. **Creating new Windows apps**
     * Enabling you to easily create new modern Windows apps with the flexibility offered by the Windows App SDK
 3. **Enabling other frameworks**
-    * Providing the native implementation for other frameworks like [React Native](https://github.com/Microsoft/react-native-windows) when running on Windows
+    * Providing the native implementation for other frameworks like [React Native](https://github.com/Microsoft/react-native-windows) when running on Windows and .NET [MAUI](https://docs.microsoft.com/dotnet/maui/what-is-maui)
```

Comment:
I think this wording could be a bit confusing. The first time I read it, I thought React Native would run on Windows and .NET MAUI, not that WinUI is powering both React Native and .NET MAUI. Maybe it could be worded in such a way that the "when running on Windows" part is at the end?

Reply:
I agree, it was a typo. Good catch!